npm-debug.log

```diff
@@ -0,0 +1,226 @@
+0 info it worked if it ends with ok
```

Comment:
Please remove this file from the last commit. This should not be under version control. I would revert the last commit with `git reset HEAD~1` and then add npm-debug.log to .gitignore. Then commit again and push force.